### ISSUE TITLE
CODEGEN: C++ text control using dot instead of ->

### ISSUE THREAD

**Issue by davavra** [wxGlade/wxGlade#498]
Two issues:

**First issue**:
C++ code generation is using dot instead of -> for wx/srchctrl
```
    text_ctrl_srch = new wxSearchCtrl(panel_3, wxID_ANY, wxEmptyString);
     : : :
    text_ctrl_srch.ShowCancelButton(true);
```
Should be:
```
    text_ctrl_srch = new wxSearchCtrl(panel_3, wxID_ANY, wxEmptyString);
     : : :
    text_ctrl_srch->ShowCancelButton(true);
```

**Second issue**:
srch ctl doesn't emit the needed includes.
Has to be added manually.
```   
#include <wx/srchctrl.h>
```

**Comment by DietmarSchwertberger** [comment]
Thanks for reporting. This should be fixed now in the repository.

**Comment by DietmarSchwertberger** [comment]
Released v1.0.3
